lib/wpt-runner.js

```diff
@@ -208,3 +218,43 @@ function recursiveReaddir(dirPath) {
     });
   });
 }
+
+function getDummyCodeOfTestDriver() {
+  return `
+(function() {
+  window.test_driver = {
+    bless: function(intent, action) {
+      return new Promise(resolve => {
+        resolve()
+      }).then(function() {
+        if (typeof action === "function") {
+         return action()
+        }
+      });
+    },
+    click: function(element) {
+      if (window.top !== window) {
+        return Promise.reject(new Error("can only click in top-level window"));
+      }
+      if (!window.document.contains(element)) {
+        return Promise.reject(new Error("element in different document or shadow tree"));
+      }
+      return new Promise(function(resolve) {
```

Comment:
`return Promise.resolve()`. Or, maybe actually implement it?

Same for all other very-long promise creations in this code.